src/routes/(app)/board/+page.svelte

```diff
@@ -8,72 +8,81 @@
 
 <PageHeader title="Styrelsen" />
 <!-- TODO: make this editable by board members with Markdown page -->
-<p>
-  Styrelsen ansvarar för den dagliga verksamheten på sektionen. Till styrelsen
-  kan du alltid vända dig om du har frågor, funderingar eller åsikter om
-  sektionen och dess verksamhet. Styrelsen sammanträder på styrelsemöten varje
-  vecka och är öppna för alla medlemmar.
-</p>
-<br />
-<p>
-  Styrelsen kan nås på <a class="text-primary" href="mailto:styrelsen@dsek.se"
-    >styrelsen@dsek.se</a
-  >
-</p>
-<br />
-<div
-  class="grid grid-cols-1 items-stretch justify-items-stretch gap-4 sm:grid-cols-2 md:grid-cols-3"
->
-  {#each data.boardPositions as member (member.position.id)}
-    <div class="card bg-base-200 transition-all">
-      {#if member.studentId && member.firstName && member.nickname && member.lastName}
-        <a
-          href="/members/{member.studentId}"
-          class="group card bg-base-200 shadow-xl transition-all hover:bg-base-200/80"
-        >
-          <figure class="mt-4 transition-transform group-hover:scale-90">
-            <MemberAvatar {member} class="h-24 w-24 rounded-full" />
-          </figure>
-          <div class="card-body px-0 text-center">
-            <h2 class="card-title mx-auto">
-              {getFullName(member)}
+<section class="mb-5 space-y-5">
+  <p>
+    Styrelsen ansvarar för den dagliga verksamheten på sektionen. Till styrelsen
+    kan du alltid vända dig om du har frågor, funderingar eller åsikter om
+    sektionen och dess verksamhet. Styrelsen sammanträder på styrelsemöten varje
+    vecka och är öppna för alla medlemmar.
```

Comment:
The lats sentence here sounds weird. It's the meetings that are open for all members, not the board it self.

Reply:
Agree. It's slightly changed now.